README.md

```diff
@@ -1 +1,3 @@
-# Hello_word
\ No newline at end of file
+# Hello_word
+
+the aim of this project is to get our pull request checked before merging
```

Comment:
- [ ] Please add README so that your repo can be professional you can refer to this [Template](https://github.com/microverseinc/readme-template)